design-documents/graph-ql/coverage/Cart.graphqls

```diff
@@ -50,7 +50,7 @@ type VirtualCartItem implements CartItemInterface @doc(description: "Virtual Car
 }
 
 type ConfigurableCartItem implements CartItemInterface {
-    child_sku: String! @doc(description: "SKU of the simple product corresponding to a set of selected configurable options.")
+    configured_variant: SimpleProduct! @doc(description: "Simple product corresponding to configured product.")
```

Comment:
I think we can unify the field name prefixes in `ConfigurableCartItem` a bit more. Having both `configured_` and `configurable_` prefixes representing some user state doesn't seem ideal.

Reply:
Agreed. I think `configurable_options` should be `configured_options`. In this we will have to deprecate and add a replacement field. I've updated the schema